includes/event-management/insert_event.php

```diff
@@ -474,9 +476,9 @@ function add_event_to_db($recurrence_arr = array()) {
 					$v = (float)preg_replace('/[^0-9\.]/ui','',$v);//Removes non-integer characters
 					$price_type = !empty($_REQUEST['price_type'][$k]) ? sanitize_text_field(stripslashes_deep($_REQUEST['price_type'][$k])) : __('General Admission', 'event_espresso');
 					$member_price_type = !empty($_REQUEST['member_price_type'][$k]) ? sanitize_text_field(stripslashes_deep($_REQUEST['member_price_type'][$k])) : __('Members Admission', 'event_espresso');
-					$member_price = !empty($_REQUEST['member_price'][$k]) ? $_REQUEST['member_price'][$k] : $v;
+					$member_price = !empty($_REQUEST['member_price'][$k]) ? sanitize_text_field($_REQUEST['member_price'][$k]) : $v;
```

Comment:
Yes, `(float)` would help in case someone input `twenty dollars` when they should have instead input `20`. Whereas `sanitize_text_field()`, not so much.

Reply:
Fixed